docs/DeveloperGuide.adoc

```diff
@@ -427,6 +427,102 @@ image::UndoRedoActivityDiagram.png[width="650" align="center"]
 ** Cons: This requires dealing with commands that have already been undone, in which we must remember to skip these commands. This violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
+//tag::Person Panel[]
+=== Person Panel
+The `PersonPanel` replaces the previous `BrowserPanel`, which is called upon using the methods in `CenterPanel`.
```

Comment:
Perhaps you can rephrase “which is called upon using the methods” to “and is called upon...”